### ISSUE TITLE
It seems the normalized dim of the feature and weight shouldn't be the same

### ISSUE THREAD

**Issue by hjy1312** [ronghuaiyang/arcface-pytorch#1]
the normalized dim of the weight should be dim 0，but that of the embedding feature should be 1，isn't it?

**Comment by hjy1312** [comment]
Sorry, i haven't notice the dim of the Linear layer is (out_features, in_features).

**Comment by xialuxi** [comment]
F.normalize(self.weight)   --------------》   F.normalize(self.weight, dim=0)
